layers/meta-balena-fsl-arm/recipes-bsp/u-boot/u-boot-boundary_%.bbappend

```diff
@@ -3,8 +3,8 @@ inherit resin-u-boot
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-# Use latest revision
-SRCREV="85f75fa9e9db29f45ef5fcee0508973f4ad82698"
+# Override bsp layer autorev and use latest revision
+SRCREV="b5b56bb07fc8f8ec541e58dc56aa48bed0ea3724"
```

Comment:
How was the BSP using autorev? I see they have it fixed also

Reply:
@floion we're using master branch: https://github.com/boundarydevices/meta-boundary/commit/1e76e65c2c22912fa99fc9f535b7e4c75b064112